snooty.toml

```diff
@@ -97,3 +97,19 @@ tls = ":abbr:`TLS (Transport Layer Security)`"
 url = ":abbr:`URL (Uniform Resource Locator)`"
 utc = ":abbr:`UTC (Coordinated Universal Time)`"
 vpc = ":abbr:`VPC (Virtual Private Cloud)`"
+
+[[banners]]
+targets = ["command/atlas-dataLakes*.txt"]
+variant = "info"
+value = """\
+    ``atlas dataLakes`` commands control the functionality previously \ 
+    named Data Lake, which is now called \
+    `Data Federation <https://www.mongodb.com/docs/atlas/data-federation/>`__ \
+    in the Atlas UI. The ``atlas dataLakes`` commands will \ 
+    continue to work for Data Federation without interruption. \ 
+    MongoDB's new \ 
+    `Data Lake <https://www.mongodb.com/docs/datalake/>`__ \ 
+    functionality doesn't include CLI commands. When we introduce a \ 
```

Comment:
When we introduce a new set of CLI commands for our new Data Lake functionality we will reach out to prevent any interruption of service for CLI users.